test/test_gipc.py

```diff
@@ -1386,6 +1386,7 @@ def signals_test_sigterm_handler():
 
 def signals_test_child_a(w):
     w.put(os.getpid())
+    w.close()
```

Comment:
Was this to fix a bug or "just" for tidiness?

Reply:
Just tidiness, it doesn't affect test execution